coverage-metrics/src/main/java/coverage_metrics/CoverageMetrics.java

```diff
@@ -15,4 +15,12 @@ public float f(int i) {
    }
    return (float)i/(k+1);
   }
+
+  public void doSomethingStupid(int max) {
+    int i;
+    float x;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "x" local variable. ([squid:S1481](http://localhost:9750/coding_rules#rule_key=squid%3AS1481))

[See it in SonarQube](http://localhost:9750/project/issues?id=org.sonarqube%3Aexample-coverage&pullRequest=36&issues=AWgTyP1d6PH_c9BW2UeW&open=AWgTyP1d6PH_c9BW2UeW)

[ISSUE_KEY]: AWgTyP1d6PH_c9BW2UeW